sql-cli/tests/test___main__.py

```diff
@@ -247,6 +247,25 @@ def test_generate_invalid(workflow_name, message, initialised_project_with_tests
     assert message in result.stdout
 
 
+def test_generate_with_output_dir(initialised_project):
```

Comment:
how about we pass the tmp_path as a fixture here and use that as the output_dir ?

Reply:
Great suggestion, @pankajkoti, particularly now that we are reusing the initialised project per test module!